validation/src/main/scala/hmda/validation/ValidationStats.scala

```diff
@@ -122,8 +140,8 @@ class ValidationStats extends HmdaPersistentActor {
         updateState(e)
       }
 
-    case AddSubmissionMacroStats(id, total, q070, q070Sold, q071, q071Sold, q072, q072Sold) =>
-      persist(SubmissionMacroStatsAdded(id, total, q070, q070Sold, q071, q071Sold, q072, q072Sold)) { e =>
+    case AddSubmissionMacroStats(id, total, q070, q070Sold, q071, q071Sold, q072, q072Sold, q075, q075Sold, q076, q076Sold) =>
+      persist(SubmissionMacroStatsAdded(id, total, q070, q070Sold, q071, q071Sold, q072, q072Sold, q075, q075Sold, q076, q076Sold)) { e =>
```

Comment:
Just curious here... It seems as though for Q075 and Q076, the only thing needed from previous years is the ratio of relevant - sold.  Would it be better to do the division in advance, and only store a single `Double`?